config/initializers/filter_parameter_logging.rb

```diff
@@ -1,8 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
-# Configure parameters to be filtered from the log file. Use this to limit dissemination of
-# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
-# notations and behaviors.
+# Configure parameters to be partially matched (e.g. passw matches password) and filtered from the log file.
+# Use this to limit dissemination of sensitive information.
+# See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += %i[
-  password passw secret token _key crypt salt certificate otp ssn
+  password passw email secret token _key crypt salt certificate otp ssn cvv cvc
```

Comment:
I've added things suggested by rails app:update and also added stuff from other apps, so no all apps should use the same logging policy for sensitive information.